diff --git a/dist/components/basic-dropdown.js b/dist/components/basic-dropdown.js
index 004f61d81d31b54e0cb30deb8342f953074072d0..189d0f94f6fad11b2cdd3ff8f7d7481c724b46a1 100644
--- a/dist/components/basic-dropdown.js
+++ b/dist/components/basic-dropdown.js
@@ -326,6 +326,7 @@ class BasicDropdown extends Component {
   }
   _getDestinationId() {
     const config = getConfig();
+    console.log(isTesting());
     if (isTesting()) {
       // document doesn't exist in fastboot apps, for this reason we need this check
       if (typeof document === 'undefined') {
